test/e2e-tests.sh

```diff
@@ -21,7 +21,7 @@ failed=0
 
 # Run upstream knative serving operator tests
 (( !failed )) && deploy_serverless_operator_latest || failed=11
-(( !failed )) && run_knative_serving_operator_tests 'master' || failed=12
+(( !failed )) && run_knative_serving_operator_tests || failed=12
```

Comment:
Should we actually run `v0.11.1` here? Or rather `release-0.11` as the checkout logic only knows branches.

Reply:
Good call. I introduced a `$KNATIVE_VERSION` var in the latest commit.

Reply:
@markusthoemmes `--branch` checkouts also tags, no problem